.travis.yml

```diff
@@ -1,43 +1,42 @@
 language: php
-
 sudo: false
+dist: trusty
+
+services:
+  - docker
+
+php:
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
-
-php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2]
-
-env:
-  global:
-    - WEBDRIVER=selenium
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.0
-      env: WEBDRIVER=selenium-remote
-      sudo: required
-      services:
-        - docker
-    - php: 5.3
-      dist: precise
-      # Force using PHP 5.6 for the test server as PHP 5.3 does not have the builtin webserver
-      env: MINK_PHP_BIN=~/.phpenv/versions/5.6/bin/php
+    - $HOME/.composer/cache
 
-before_script:
-  - sh bin/run-"$WEBDRIVER".sh
-
-  - composer install
+install:
+  - travis_retry composer self-update
+  - travis_retry composer update --no-interaction $DEPENDENCIES
```

Comment:
1. what `$DEPENDENCIES` do considering that it isn't defined?
2. what are the benefits compared to `composer install` command, that is replaced by this command?

Reply:
`travis_retry` is just an retry mechanism, returned `composer install`